engines/engine.cpp

```diff
@@ -454,25 +454,39 @@ void GUIErrorMessageFormat(Common::U32String fmt, ...) {
 	GUIErrorMessage(msg);
 }
 
-void Engine::checkCD() {
-#ifdef USE_VORBIS
+/**
+ * Checks if supported (extracted) audio files are found.
+ *
+ * @return			true if audio files of the expected naming scheme are found, as long as ScummVM
+ *					is also build with support to the respective audio format (eg. ogg, flac, mad/mp3)
```

Comment:
```suggestion
 *					is also built with support to the respective audio format (eg. ogg, flac, mad/mp3)
```

Reply:
Fixed in https://github.com/scummvm/scummvm/pull/3018/commits/45acce61e2002b1f0f6bea5562ca93dd1709a350